testing/config/settings/base.py

```diff
@@ -69,7 +69,8 @@
 
 PROJECT_SEND_INVITE = True
 
-PROJECT_REMOVE_VIEWS = True
+PROJECT_VIEWS_SYNC = True
```

Comment:
Depending on the default settings and the tests, maybe this should be False (or removed from this file).